### ISSUE TITLE
Feature request: Grid SetPage & SetFilters

### ISSUE THREAD

**Issue by CSam1989** [vikramlearning/blazorbootstrap#945]
**Describe the feature**
It would be much appreciated to add methods in BlazorBootstrap.Grid to update page number & filters programmatically (perhaps a way to update GridSettings).

I could not find a way to update these values.

**Versions (please complete the following information):**
 - .NET Version: NET 8
 - BlazorBootstrap: 3.3.1
 - Blazor Server
 - Blazor Interactive Render Mode: Server
 - Blazor Interactivity Location: Global

**Comment by kellcomnet2** [comment]
Agreed, the ability to programmatically set filters is greatly missed

**Comment by Cybil74** [comment]
As for me, I have a need for something like a "Grid.GoToPage(int pageNumber)" method. I have a scenario where I'd like, immediately after page load, to sort a paged grid ascending and, at the same time, to see the most recent rows, i.e. to go to the last page immediately after page load. In the absence of an "Grid.GoToPage()" method , in order to see the most recent rows on page load, the only alternative I have is to sort descending, which is something I don't want to do.
